### PR TITLE
Statistics Device

### PR DESCRIPTION
**Pull request by kayoub5**
This is the last nail in the coffin of unifying the Libpcap namespaces across all platforms, no more `Npcap` or `Winpcap` namespaces.

### CODE REVIEW

**Comment by kayoub5** [comment]
@chmorgan waiting for your review

**Review by chmorgan — CHANGES_REQUESTED** [review]
Added one comment, the other thing I noticed was that code coverage went down a little bit with the change.

File: Test/RemotePcapTests.cs [hunk 1]
```diff
@@ -72,25 +73,18 @@ public void PwdAuthTest()
...
-                    var npcapDevices = NpcapDeviceList.Devices(IPAddress.Loopback, NpcapDeviceList.RpcapdDefaultPort, goodCred);
-                    CollectionAssert.IsNotEmpty(npcapDevices);
-
-                    var devices = new PcapDevice[]{
-                        // using NpcapDevice
-                        npcapDevices[0],
-                        // using rpcap with LibPcapLiveDevice should be possible
-                        new LibPcapLiveDevice(pcapIfs[0])
-                    };
-                    foreach (var device in devices)
```
[the remark this answers is not in the document]
Comment (kayoub5) [reply Test/RemotePcapTests.cs:84]: `NpcapDevice` is no more, so we only have `LibPcapLiveDevice` left, thus the need for a foreach loop is no more.

**Comment by kayoub5** [comment]
@chmorgan Coverage went down, since I removed the Npcap code, and the statistic code I put have fewer lines.
Check codecov report, to double check if I missed a method that should be covered.

**Comment by chmorgan** [comment]
@kayoub5 yeah it looks like the coverage is good. Its an interesting thing to indicate a negative value when we are taking out code...

**Review by chmorgan — APPROVED** [review]

**Pull request merged**